src/http/zcl_abapgit_proxy_config.clas.abap

```diff
@@ -81,4 +94,14 @@ CLASS ZCL_ABAPGIT_PROXY_CONFIG IMPLEMENTATION.
         cv_proxy_url = rv_proxy_url ).
 
   ENDMETHOD.
+
+
+  METHOD bypass_proxy.
+
+    IF iv_repo_url IN mo_settings->get_proxy_bypass( ).
```

Comment:
I haven't looked into this in detail but won't and empty range table cause every URL to be included in the proxy bypass?

Reply:
I can confirm this, an empty range would cause this to always evaluate to `true`

Reply:
Yes, you're right. Rookie mistake...

I'll fix it.